google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/AbstractITDatastoreTest.java

```diff
@@ -1316,6 +1319,12 @@ public void testUpdate() {
     datastore.delete(ENTITY3.getKey());
   }
 
+  private void assertDatastoreException(
+      DatastoreException expected, String reason, int datastoreStatusCode) {
```

Comment:
nit: this naming feels backward to me. I interpret expected as the value that is expected to hold in the assertion, not the exception that was expected to be thrown.

May not be worth the hassell, but I prefer the following pattern for asserting exceptions that are expected to be thrown:
```java
DatastoreException datastoreException = org.junit.Assert.assertThrows(DatastoreException.class, () -> datastore.update(ENTITY3));
com.google.common.truth.Truth.assertThat(datastoreException.getReason()).isEqualTo(NOT_FOUND.name());
com.google.common.truth.Truth.assertThat(datastoreException.getCode()).isEqualTo(5);
```

Reply:
I like it! I've made these updates.